python/tvm/relay/analysis.py

```diff
@@ -408,3 +408,17 @@ def structural_hash(value):
         msg = ("found value of type {0} expected" +
                "relay.Expr or relay.Type").format(type(value))
         raise TypeError(msg)
+
+
+def extract_fused_functions():
```

Comment:
API style: the analysis API so far are not functors(unlike the Pass in the transform namespace). So we should directly take a IRModule and return a collection of functions for now.